crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -497,27 +518,38 @@ impl BladeRenderer {
             };
 
             let vertex_buf = unsafe { self.instance_belt.alloc_typed(&vertices, &self.gpu) };
-            let mut pass = self.command_encoder.render(
-                "paths",
+            let frame_view = tex_info.raw_view;
+            let render_target = if let Some(msaa_view) = tex_info.msaa_view {
                 gpu::RenderTargetSet {
```

Comment:
this should be slightly different
```rust
let color_target = if let Some(msaa_view) = tex_info.msaa_view {
gpu::RenderTarget {
                        view: tex_info.raw_view,
                        view: msaa_view,
                        init_op: gpu::InitOp::Clear(gpu::TextureColor::OpaqueBlack),
                        finish_op: gpu::FinishOp::ResolveTo(frame_view),
                    }
} else {
gpu::RenderTarget {
                        view: frame_view,
                        init_op: gpu::InitOp::Clear(gpu::TextureColor::OpaqueBlack),
                        finish_op: gpu::FinishOp::Store,
                    }
}
```

Reply:
Do you means should rename `render_target` to `color_target`? Or is other I missed?

Reply:
Your `render_target` is really a `RenderTargetSet`. I'm just suggesting to instead do if/else on `gpu::RenderTarget` inside it

Reply:
I see. I didn't notice this before, there is duplicate code